src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Largest Contentful Paint
+subhead: An important, new user-centric performance metric
```

Comment:
Is there a way to rephrase this so the person understand what the metric will help them with?

Reply:
Updated to: "Making it easier to know when a page's important content has loaded"